data-sources/audit-prometheus-data-ingest-analysis/config.yml

```diff
@@ -0,0 +1,8 @@
+id: audit-prometheus-data-ingest-analysis
+displayName: Prometheus OpenMetrics Integration Stats
+description: Paginated dashboard that analyzes overall data ingest generated by the New Relic Prometheus OpenMetrics Integration (aka POMI) and outlines options available to fine-tune your environment to help control ingest and cost.
```

Comment:
This description speaks to the dashboard, lets have it speak more to the data source.
The resource it links to is a doc that describes the different ways you can monitor prometheus, for which we have core data sources for each. 

The description though is about looking at data generated from POMI, so we should probably instead remove this and assign the quickstart to the core pomi datasource